src/__tests__/components/MultiRowField/MultiRowField.test.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { reduxForm } from 'redux-form';
+import { createStore } from 'redux';
+import { connect, Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import toJSON from 'enzyme-to-json';
+
+import MultiRowField from '../../../components/MultiRowField/MultiRowField';
+
+const initialValues = {
+    'mock-name': [
+        {
+            name: 'me',
+        },
+        {
+            name: 'you',
+        },
+    ]
+};
+
+const store = createStore(() => ({
+    form: {
+        mockForm: {
+            values: initialValues,
+        }
+    }
+}));
+
+const Form = ({ children, initialValues }) => children;
+
+const ConnectedForm = connect(
```

Comment:
Could you create a ticket for extracting this logic later on when you need to create similar tests for other fields? That way you won't forget :)

Reply:
Why leave for tomorrow the stuff, which could be done today?)
Done, caballero 😉